flytekit/types/structured/basic_dfs.py

```diff
@@ -35,6 +36,54 @@ def get_storage_options(cfg: DataConfig, uri: str, anon: bool = False) -> typing
     return None
 
 
+class PandasToCSVEncodingHandler(StructuredDatasetEncoder):
+    def __init__(self):
```

Comment:
You need to register the handler in this [file](https://github.com/flyteorg/flytekit/blob/a1636a89daee2301a056b3eba41fe4dcea1257b7/flytekit/types/structured/__init__.py#L27-L34)